src/domain/common/whiteboard/whiteboard.resolver.fields.ts

```diff
@@ -63,4 +66,24 @@ export class WhiteboardResolverFields {
   ): Promise<IProfile> {
     return loader.load(whiteboard.id);
   }
+
+  @ResolveField(() => Boolean, {
+    nullable: false,
+    description:
+      'Indicates whether guest collaborators are currently allowed via GLOBAL_GUEST permissions.',
+  })
+  async guestContributionsAllowed(
+    @Parent() whiteboard: IWhiteboard
+  ): Promise<boolean> {
+    if (!whiteboard.authorization) {
+      throw new EntityNotInitializedException(
+        `Authorization not initialized for whiteboard: ${whiteboard.id}`,
```

Comment:
We have an agreement to not include dynamic data in the exception message.
Pass variables in the `details` parameter of the exception.